.github/workflows/benchmark-nodes-base.yml

```diff
@@ -0,0 +1,47 @@
+name: Benchmark nodes-base
+
+on:
+  pull_request:
+    paths:
+      - 'packages/cli/**'
```

Comment:
should we remove `packages/cli` from this list for now? changes to that package would not affect nodes, at least for how the nodes tests are currently setup.